interval-functor.cabal

```diff
@@ -1,7 +1,7 @@
 cabal-version:       2.2
 
 name:                interval-functor
-version:             0.0.0.0
+version:             0.1.0.0
```

Comment:
I don’t think this constitutes a major release; there’s no backwards-compat issue that I can see.

Reply:
New instances being exported may conflict with orphan instances defined in user code.

Reply:
You could make the same claim for _any_ added symbol—it could break a build if you’re importing it open in a module using a symbol of the same name from elsewhere.

Reply:
Other sorts of names are safe to add assuming the consumers follow the advised practice of importing qualified or with explicit import lists. On the contrary, the import of classes is not possible to opt out.

Reply:
Actually, [the official guide for PVP][1] says that added non-orphan instances are not considered a breaking change. These instances here are non-orphan. So you are right.

[1]: https://pvp.haskell.org/